src/store.c

```diff
@@ -49,11 +52,13 @@ struct rill_packed header
     uint64_t keys;
     uint64_t pairs;
 
-    uint64_t vals_off;
     uint64_t data_off;
```

Comment:
If we're committing to the col a/b nomenclature now (I was going to do it after as there's a lot of renaming to do), then we should probably be consistent about it and name it `data_a_off` and `data_b_off`. Another alternative worth considering, `data_off[2];` and using the col enum to access the column you want.

Reply:
Fixorzed. We check revisit the array version later.